From 65e10b1ecbdc2dc86b1769aed567d1edcf7f3e38 Mon Sep 17 00:00:00 2001
From: Evan Maddock <maddock.evan@vivaldi.net>
Date: Tue, 10 Aug 2021 17:06:54 -0400
Subject: [PATCH 1/1] Change from Fusion to Native system style.

Credit to Yurizal Susanto <rizal.sagi@gmail.com> for the original patch; it had to be rebased as upstream changed the styling preferences.

Signed-off-by: Evan Maddock <maddock.evan@vivaldi.net>
---
 src/tiled/preferences.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tiled/preferences.cpp b/src/tiled/preferences.cpp
index 11b1dda0..e45a6cae 100644
--- a/src/tiled/preferences.cpp
+++ b/src/tiled/preferences.cpp
@@ -244,7 +244,7 @@ Preferences::ApplicationStyle Preferences::applicationStyle() const
 #if defined(Q_OS_MAC)
     return static_cast<ApplicationStyle>(get<int>("Interface/ApplicationStyle", SystemDefaultStyle));
 #else
-    return static_cast<ApplicationStyle>(get<int>("Interface/ApplicationStyle", TiledStyle));
+    return static_cast<ApplicationStyle>(get<int>("Interface/ApplicationStyle", SystemDefaultStyle));
 #endif
 }
 
-- 
2.32.0

